src/components/Footer/sgds-footer.ts

```diff
@@ -51,8 +90,8 @@ export class SgdsFooter extends SgdsElement {
           <div class="container-fluid">
             <div class="row footer-header">
               <div class="col col-lg-6 col-md-12">
-                <div class="title">${this.title}</div>
-                <div class="description">${this.description}</div>
+                <div class="title">${this.title ? this.title : "Footer title"}</div>
```

Comment:
is this a guideline that must fill in Footer title or description? if the user doesnt want to put title or description then how?

Reply:
Nope there is no guideline that user need to fill up the description. 
Resolved in [d8230f6](https://github.com/GovTechSG/sgds-web-component/pull/61/commits/d8230f6c787ccec60641646b1c1df5df0a2f6020)